.travis.yml

```
@@ -18,6 +18,7 @@ services:
 install:
 
   - pip install -r requirements.txt
+  - pip install requests==2.19.1
```

```suggestion
  - pip install requests==2.19.1
```

Not sure if this is needed or should be elsewhere.